_tuning-your-cluster/availability-and-recovery/segment-replication/index.md

```diff
@@ -101,11 +109,11 @@ When using segment replication, consider the following:
 1. Enabling segment replication for an existing index requires [reindexing](https://github.com/opensearch-project/OpenSearch/issues/3685).
 1. [Cross-cluster replication](https://github.com/opensearch-project/OpenSearch/issues/4090) does not currently use segment replication to copy between clusters.
 1. Segment replication is not compatible with [document-level monitors]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/api/#document-level-monitors), which are used with the [Alerting]({{site.url}}{{site.baseurl}}/install-and-configure/plugins/) and [Security Analytics]({{site.url}}{{site.baseurl}}/security-analytics/index/) plugins. The plugins also use the latest available data on replica shards when using the `immediate` refresh policy, and segment replication can delay the policy's availability, resulting in stale replica shards.
-1. Segment replication leads to increased network congestion on primary shards. See [Issue - Optimize network bandwidth on primary shards](https://github.com/opensearch-project/OpenSearch/issues/4245).
-1. Integration with remote-backed storage as the source of replication is [currently not supported](https://github.com/opensearch-project/OpenSearch/issues/4448). 
-1. Read-after-write guarantees: Segment replication does not currently support setting the refresh policy to `wait_for`.  If you set the `refresh` query parameter to `wait_for` and then ingest documents, you'll get a response only after the primary node has refreshed and made those documents searchable. Replica shards will respond only after having written to their local translog. We are exploring other mechanisms for providing read-after-write guarantees. For more information, see the corresponding [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/6046).  
-1. System indexes will continue to use document replication internally until read-after-write guarantees are available. In this case, document replication does not hinder the overall performance because there are few system indexes.
-   
+1. Segment replication leads to increased network congestion on primary shards when not using a remote store. With a remote store, the primary shard can upload segments to the remote store, then you can down replicas from the same store. 
+1. Read-after-write (RAW) guarantees: As of OpenSearch 2.10, RAW guarantees are supported when using [`get`]({{site.url}}{{site.baseurl}}/api-reference/document-apis/get-documents/) or [`mget`]({{site.url}}{{site.baseurl}}/api-reference/document-apis/multi-get/) API operations, or through searching with `_primary`.
+1. As of OpenSearch 2.10, system indexes are now supported inside segment replication. 
+1. Get, MultiGet, TermVector, and MultiTermVector requests serve strong reads by routing requests to the primary shards. In a read heavy cluster, we recommend setting the `realtime` parameter in these requests to `false`, especially with listed request types.
```

Comment:
I need a little more context to address this comment.

Reply:
Routing Get, mGet and termVector requests to primary could hurt performance in read heavy cases or if primary is overloaded. Since there will be more requests to be handled by primary shards instead of distributing this across primary and replica shards. For more details: https://github.com/opensearch-project/OpenSearch/issues/8700

Reply:
Awesome! Thank you!